lib/tasks/scraper.rake

```diff
@@ -0,0 +1,9 @@
+task queue_logs: :environment do
```

Comment:
@speedywizard @AndriiMysko why we need this rake task?
You can use `sidekiq-scheduler` instead